src/features/analytics/components/IndicatorComponent/IndicatorComponent.css

```diff
@@ -0,0 +1,166 @@
+.absolute-component,
+.marker-list,
+.single-component,
+.list-component {
+  position: relative;
+  border-radius: 10px;
+  width: 100%;
+  height: auto;
+  box-shadow: 0 0 43px rgb(24 14 54 / 20%);
+  background: #fff;
```

Comment:
background - this is a whole set of properties
it is better to use a specific property "background-color" so that the browser does not check the other properties